tests/test_datasets.py

```diff
@@ -1,74 +1,108 @@
 import sys
+from dataclasses import dataclass
 from unittest import mock
 
 import numpy as np
 import pandas as pd
 import pytest
 from numpy.testing import assert_array_equal
 
-from sknnr.datasets import load_moscow_stjoes
+from sknnr.datasets import load_moscow_stjoes, load_swo_ecoplot
 
 
-def test_load_moscow_stjoes():
+@dataclass
+class DatasetConfiguration:
+    load_function: callable
+    n_samples: int
+    n_targets: int
+    n_features: int
+
+
+CONFIGURATIONS = [
+    DatasetConfiguration(
+        load_function=load_moscow_stjoes, n_samples=165, n_targets=35, n_features=28
+    ),
+    DatasetConfiguration(
+        load_function=load_swo_ecoplot, n_samples=3005, n_targets=25, n_features=18
+    ),
+]
+
+CONFIGURATION_IDS = ["moscow_stjoes", "swo_ecoplot"]
```

Comment:
Can we combine these into a dict with the IDs as keys and configs as values, just to ensure they stay synced?

Reply:
Perfect.  I've taken a stab at this.